plugins/module_utils/kibana.py

```diff
@@ -204,6 +224,51 @@ def get_pkg_policy(self,integration_name, agent_policy_id):
   
   def create_pkg_policy(self,pkg_policy_name, pkg_policy_desc, agent_policy_id, integration_object, namespace="default"):
     pkg_policy_object = self.get_pkg_policy(integration_object['name'],agent_policy_id)
+    inputs_body = []
+    if 'policy_templates' in integration_object:
+      for policy_template in integration_object['policy_templates']:
+        if 'inputs' in policy_template:
+          if policy_template['inputs'] != None:
+            for policy_input in policy_template['inputs']:
+              inputs_body_entry = {}
+              inputs_body_entry['policy_template'] = policy_template['name']
+              inputs_body_entry['enabled'] = True
+              inputs_body_entry['type'] = policy_input['type']
+              inputs_body_entry['config'] = {}
+              input_body_template_var = {}
+              if 'vars' in policy_input:
+                for policy_template_var in policy_input['vars']:
+                  if 'value' in policy_template_var:
+                    input_body_template_var[policy_template_var['name']] = { "type": policy_template_var['type'], "value": policy_template_var['value']}
+                  else:
+                    input_body_template_var[policy_template_var['name']] = { "type": policy_template_var['type']}
+              inputs_body_entry['vars'] = input_body_template_var
+              inputs_body_streams = []
+              for integration_object_input in integration_object['data_streams']:
+                inputs_body_stream_entry = {}
+                #inputs_body_stream_entry['enabled'] = True
+                for integration_input_stream in integration_object_input['streams']:
+                  if 'enabled' in integration_input_stream:
+                    inputs_body_stream_entry['enabled'] = integration_input_stream['enabled']
+                  else:
+                    inputs_body_stream_entry['enabled'] = False
+                  if integration_input_stream['input'] == policy_input['type']:
+                    inputs_body_streams_datastream = {}
+                    inputs_body_streams_datastream['dataset'] = integration_object_input['dataset']
+                    inputs_body_streams_datastream['type'] = integration_object_input['type']
+                    inputs_body_stream_entry['data_stream'] = inputs_body_streams_datastream
+                    input_body_stream_var = {}
+                    for integration_stream_var in integration_input_stream['vars']:
+                      try: 
+                        input_body_stream_var[integration_stream_var['name']] = { "type": integration_stream_var['type'], "value": integration_stream_var['default']}
+                      except:
```

Comment:
We should refrain from using bare excepts. This should be replaced by checking if the 'default' key is present in `integration_stream_var` OR catching the error specifically as a `KeyError`.